src/app/plugins/platform/Docker/plugins/cms/Drupal8/index.ts

```diff
@@ -387,6 +442,21 @@ class Drupal8 extends Generator {
     );
   }
 
+  async install(): Promise<void> {
+    if (!this.options.skipInstall) {
```

Comment:
Very light opinion nit... If I can I always make if conditions without negation, and return early.

```
if (this.options.skipInstall) {
  this.debug(
    'Skipping final Composer installation due to `--skip-install` option.',
  );
  return;
}

// Run final installation of all Composer dependencies now that all
```

Reply:
I hadn't done this since it's within one of the main lifecycle methods (`install()`), but you make a good point for readability the way it sits now with that being the only action in the method. I've applied this in f13dfd5.